test/Microsoft.Framework.Configuration.Ini.Test/IniConfigurationSourceTest.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Framework.Configuration.Ini.Test
 {
-    public class IniConfigurationSourceTest
+    public class IniConfigurationProviderTest
```

Comment:
For the sake of consistency, rename this class like `EnvironmentVariablesTest` or rename the others one to be like this.